packages/core-utils/src/index.ts

```diff
@@ -22,4 +23,5 @@ const core = {
   ui
 };
 
+export { SafeSuspense };
```

Comment:
Note that this is to accommodate tests run locally only, and that no release will be triggered for this package.